server/service/integration_mdm_lifecycle_test.go

```diff
@@ -674,15 +677,21 @@ func (s *integrationMDMTestSuite) TestLifecycleSCEPCertExpiration() {
 		}
 	}
 
+	// run the worker to complete enrollment and get any worker-generated commands
+	s.runWorker()
```

Comment:
I couldn't quite fix it yet, this is a WIP, but it was working most of the time because the worker did not run. When it failed, it was because the worker cron job did run just before the assertion at line 810 (original code) that asserts that there's no command pending, but there is one - the install of fleetd.

My understanding of SCEP renewal (which is limited) is that there SHOULDN'T be any command pending - so the failure is an indication that something is wrong IIUC, but it might be that the test does not set things up correctly, not necessarily that the renewal itself has bugs.